app/services/ccms/requestors/case_add_requestor.rb

```diff
@@ -123,7 +123,7 @@ def generate_case_docs(xml)
       end
 
       def generate_other_parties(xml)
-        generate_opponent(xml, @legal_aid_application.opponent)
+        generate_opponent(xml, @legal_aid_application.opponents.first)
```

Comment:
With limited knowledge of the payload, i may be talking squat, but as there can be multiple opponents, should this be an array?

Reply:
This looks like a rebase issue to me - I am going to return the PR to draft for now and address all of your -very valid- points!

Reply:
Yeah, a later commit had resolved this 😕 
Not sure how this was in the review state 😖